03-specifying-sample-designs.Rmd

```diff
@@ -380,35 +382,33 @@ Replicate weights are often included on analysis files instead of, or in additio
 
 The different types of replicate weights largely differ in step 1 - how the sample is divided into subsamples, and step 4 - which multiplication factors (scales) are used to multiply the variance.
 
-### BRR method
+### Balanced Repeated Replication (BRR) Method
 
-The BRR method requires a stratified sample design with two PSUs in each stratum. Each replicate is constructed by deleting one PSU per stratum using a Hadamard matrix. For the PSU that is included, the weight is generally multiplied by two but may have other adjustments, such as post-stratification. A Hadamard matrix is a special square matrix with entries of +1 or -1 with mutually orthogonal rows. Hadamard matrices must have one row, two rows, or a multiple of four rows. An example of a $4\times4$ Hadamard matrix is below:
+The BRR method requires a stratified sample design with two PSUs in each stratum. Each replicate is constructed by deleting one PSU per stratum using a Hadamard matrix. For the PSU that is included, the weight is generally multiplied by two but may have other adjustments, such as post-stratification. A Hadamard matrix is a special square matrix with entries of +1 or -1 with mutually orthogonal rows. Hadamard matrices must have one row, two rows, or a multiple of four rows. An example of a $4\times4$ Hadamard matrix is below: 
```

Comment:
This may be a stupid question, but if I have it, then we may have a reader or two that has it as well :) 

How do you determine the size of the Hadamard matrix for this?

Reply:
It is the multiple of 4 that is just bigger than the number of PSUs.